iceoryx2-bb/log/src/lib.rs

```diff
@@ -185,6 +188,34 @@ pub enum LogLevel {
     Fatal = 5,
 }
 
+impl LogLevel {
+    fn from_str_fuzzy(s: &str) -> LogLevel {
+        match s.to_lowercase().as_str() {
+            "trace" => LogLevel::Trace,
+            "debug" => LogLevel::Debug,
+            "info" => LogLevel::Info,
+            "warn" => LogLevel::Warn,
+            "error" => LogLevel::Error,
+            "fatal" => LogLevel::Fatal,
+            _ => {
+                println!("Error: unknown logging level {:?}", s);
+                println!("Warning: setting log level as : Info");
+                DEFAULT_LOG_LEVEL
```

Comment:
```suggestion
                println!("Warning: setting log level as : {}", DEFAULT_LOG_LEVEL);
                DEFAULT_LOG_LEVEL
```
I hope this works :sweat_smile: 
If not, then you would need to implement `Display`

Reply:
@elBoberido using `{:?}` for printing `DEFAULT_LOG_LEVEL` worked

```
Error: unknown logging level "BlaBla"
Warning: setting log level as : Info
Trigger event with id 1 ...
```

Reply:
@elBoberido makes sense, I changed it to 
```
println!(
                    "Invalid value for 'IOX_LOG_LEVEL' environment variable!\
                \nFound: {:?}\
                \nAllowed is one of: fatal, error, warn, info, debug, trace\
                \nSetting log level as : {:?}",
                    s, DEFAULT_LOG_LEVEL
                );
```